bears/ruby/RuboCopBear.py

```diff
@@ -28,8 +28,188 @@ def create_arguments(filename, file, config_file, rubocop_config: str=""):
         args = (filename, '--stdin', '--format=json')
         if rubocop_config:
             args += ('--config', rubocop_config)
+        else:
+            args += ('--config', config_file)
         return args
 
+    @staticmethod
+    def generate_config(filename, file,
+                        access_modifier_indentation: str='indent',
+                        preferred_alias: str='prefer_alias',
+                        align_hash_rocket_by: str='key',
+                        align_colon_by: str='key',
+                        inspect_last_argument_hash: str='always_inspect',
+                        align_parameters: str='with_first_parameter',
+                        class_check: str='is_a?',
+                        comment_keywords: tuple=("TODO",
+                                                 "FIXME",
+                                                 "OPTIMIZE",
+                                                 "HACK",
+                                                 "REVIEW"),
+                        min_if_unless_guard: int=1,
+                        indentation_width: int=2,
+                        method_name_case: str='snake_case',
+                        string_literals: str='single_quotes',
+                        variable_name_case: str='snake_case',
+                        max_class_length: int=100,
+                        class_length_count_comments: bool=False,
+                        max_module_length: int=100,
+                        module_length_count_comments: bool=False,
+                        cyclomatic_complexity: int=6,
+                        max_line_length: int=80,
+                        line_length_allow_here_doc: bool=True,
+                        line_length_allow_uri: bool=True,
+                        max_method_length: int=10,
+                        method_count_comments: bool=False,
+                        max_parameters: int=5,
+                        count_keyword_args: bool=True,
+                        ignore_unused_block_args_if_empty: bool=True,
+                        allow_unused_block_keyword_arguments: bool=False,
+                        ignore_unused_method_args_if_empty: bool=True,
+                        allow_unused_method_keyword_args: bool=False):
+        """
+        Not all settings added.
+        Notable settings missing: Rails settings.
+
+        :param access_modifier_indentation:
+            Indent private/protected/public as deep as method definitions
+            options:
+                ``indent`` :  Indent modifiers like class members.
+                ``outdent`` : Indent modifiers one level less than
+                              class members.
+        :param preferred_alias:
+            Which method to use for aliasing in ruby?
```

Comment:
most of the stuff with question marks was documented like that, but i think we need  to make it better anyways , here's the rubocops [defautlt.yml](https://github.com/bbatsov/rubocop/blob/master/config/default.yml)